components/changelog/ChangeComponent.tsx

```diff
@@ -8,21 +8,35 @@ const ChangeComponent = (props: Change) => {
         <li className={'py-3 sm:py-4'}>
             <div className={'flex items-center space-x-4'}>
                 <div className={'shrink-0'}>
-                    <FontAwesomeIcon className={'text-xl'} icon={determineIcon(category)}></FontAwesomeIcon>
+                    <FontAwesomeIcon className={`text-xl ${determineClassNameIcon(category)}` icon={determineIcon(category) }></FontAwesomeIcon>
```

Comment:
```suggestion
                    <FontAwesomeIcon className={`text-xl ${determineClassNameIcon(category)}`} icon={determineIcon(category)}></FontAwesomeIcon>
```

This suggestion should fix it.